ckan/config/deployment.ini_tmpl

```diff
@@ -47,6 +47,9 @@ who.log_file = %(cache_dir)s/who_log.ini
 ## Database Settings
 sqlalchemy.url = postgresql://ckan_default:pass@localhost/ckan_default
 
+## Migration Settings
```

Comment:
This is an obscure setting. How about sticking this more round L99?

Reply:
Yep, that sounds logical - just pushed a commit